index.js

```diff
@@ -4,7 +4,7 @@ import MochaAdapter from './lib/adapters/MochaAdapter.js'
 import TapReporter from './lib/reporters/TapReporter.js'
 import ConsoleReporter from './lib/reporters/ConsoleReporter.js'
 import {Assertion, Test, Suite} from './lib/Data.js'
-import {autoRegister} from './lib/helpers.js'
```

Comment:
Although every function will be [exported individually](https://github.com/js-reporters/js-reporters/blob/349292be2e8e115b71b476b487897080055a93c4/index.js#L18).